exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -190,8 +223,59 @@ public void addSummary(TableBuilder tb, HashMap<String, Long> majorFragmentBusyT
     tb.appendFormattedInteger(recordSum);
 
     final ImmutablePair<OperatorProfile, Integer> peakMem = Collections.max(opList, Comparators.operatorPeakMemory);
-    tb.appendBytes(Math.round(memSum / size));
-    tb.appendBytes(peakMem.getLeft().getPeakLocalMemoryAllocated());
+
+    //Inject spill-to-disk attributes
+    Map<String, String> avgSpillMap = null;
+    Map<String, String> maxSpillMap = null;
+    if (hasSpilledToDisk) {
+      avgSpillMap = new HashMap<>();
+      //Average SpillCycle
+      float avgSpillCycle = (float) spillCycleSum/size;
+      avgSpillMap.put(HTML_ATTRIB_TITLE, DECIMAL_FORMATTER.format(avgSpillCycle) + " Spills on average");
```

Comment:
I think `s` can be in lower case: `spills on average`.

Reply:
Agreed. Will fix it.